tests/storage/glusterfs/conftest.py

```diff
@@ -1,7 +1,7 @@
 import logging
 import pytest
-
```

Comment:
Do not remove this empty line.

Reply:
why ?
do we need to have an empty line between import and from lines ?

Reply:
IMHO yes, this ease readability.

Reply:
ok, i will change it. I didn't know, as it wasn't a comment on the iso pr ;)

Reply:
Second answer to "why": because it is not related to the objective of your commit. We don't want to mix unrelated changes together. Formatting changes that you want to do to would belong to a separate commit (and a different PR usually), if relevant.

Reply:
i don't want to do any formatting change... i just keep writing as i do with the iso pr for that part, so i just delet an empty line when i add an import one. But the explanation from Benjamin is enough. I need to have that empty line, si i will put it back. And i won't change something else in this PR.